spec/unit/default_spec.rb

```diff
@@ -30,30 +31,36 @@
   it 'checks out test_a and test_b' do
     expect(chef_run).to sync_git('/opt/custom').with(
       repository: 'https://github.com/osuosl/nodejs-test-apps.git',
-      branch: 'custom')
+      branch: 'custom'
+    )
```

Comment:
You need to check if its going to use the correct user and group in the test here.